libraries/AC_AttitudeControl/AC_AttitudeControl.h

```diff
@@ -389,9 +390,20 @@ class AC_AttitudeControl {
     // get the value of the angle P scale that was used in the last loop, for logging
     const Vector3f &get_angle_P_scale_logging(void) const { return _angle_P_scale_used; }
     
+    // setup a one loop PD scale multiplier, multiplying by any
+    // previously applied scale from this loop. This allows for more
+    // than one type of scale factor to be applied for different
+    // purposes
+    void set_PD_scale_mult(const Vector3f &angle_P_scale) { _pd_scale *= _pd_scale; }
```

Comment:
Looks like a little bug here - not using the argument that's being passed in.  I wonder if we need to add any checks to this to be sure that invalid values aren't passed in?  This part is optional.

Reply:
Fixed thanks